in-app-purchase/src/main/java/com/rakuten/tech/mobile/miniapp/iap/InAppPurchaseExecutor.kt

```diff
@@ -0,0 +1,122 @@
+package com.rakuten.tech.mobile.miniapp.iap
+
+import android.app.Activity
+import android.util.Log
+import com.android.billingclient.api.BillingClient
+import com.android.billingclient.api.BillingClientStateListener
+import com.android.billingclient.api.BillingFlowParams
+import com.android.billingclient.api.BillingResult
+import com.android.billingclient.api.ConsumeParams
+import com.android.billingclient.api.Purchase
+import com.android.billingclient.api.PurchasesUpdatedListener
+import com.android.billingclient.api.SkuDetails
+import com.android.billingclient.api.SkuDetailsParams
+
+/**
+ * The In-App purchase executor which acts as a default implementation of InAppPurchaseProvider.
+ * @param context should use the same activity context for #MiniAppDisplay.getMiniAppView.
+ */
+class InAppPurchaseExecutor(private val context: Activity) : InAppPurchaseProvider {
```

Comment:
Rename `InAppPurchaseExecutor` to `IAPBridgeDispatcherDefault` as it is the default implementation of `IAPBridgeDispatcher`

Reply:
Yeah, this name is kinda confusing, so I have renamed to `InAppPurchaseProviderDefault` to be matched with the interface's name.